ppdc/ppdc-string.cxx

```diff
@@ -25,13 +25,13 @@ ppdcString::ppdcString(const char *v)	// I - String
 
   if (v)
   {
-    size_t vlen = strlen(v);
-
-    value = new char[vlen + 1];
-    memcpy(value, v, vlen + 1);
+    size_t vlen = strlen(v) + 1;
+    value = static_cast<char *>(malloc(vlen));
```

Comment:
OK, so *if* we change this code at all, just use `strdup`.

Reply:
I was thinking about that, though it is BSD/Mac only it seems